### PR TITLE
Mage_Sendfriend - DOC block update

### PR DESCRIPTION
**Pull request by sreichel**
- doc blocks added/fixed
- PSR2 fixes (whitespaces, linebreaks, ...)

### CODE REVIEW

**Comment by sreichel** [comment]
Bump. :)

File: app/code/core/Mage/Sendfriend/Model/Sendfriend.php [hunk 1]
```diff
@@ -177,7 +187,7 @@ public function validate()
         }
 
         $email = $this->getSender()->getEmail();
-        if (empty($email) OR !Zend_Validate::is($email, 'EmailAddress')) {
+        if (empty($email) or !Zend_Validate::is($email, 'EmailAddress')) {
```
Comment (tmotyl) [inline app/code/core/Mage/Sendfriend/Model/Sendfriend.php:190]: This should be ||

Comment (tmotyl) [reply app/code/core/Mage/Sendfriend/Model/Sendfriend.php:190]: Or create an issue for it and we will rector all places like that in one go

**Comment by sreichel** [comment]
> 
> 
> I approve, but please create an issue for inconsisten and/or vs || &&

I guess are the are many occurrences ... i'll create a tickt for.

**Pull request merged**